sig-network/concepts-and-designs/kubesphere-network-policy.md

```diff
@@ -48,23 +45,47 @@ spec:
         - namespaceSelector:
             matchLabels:
               "kubesphere.io/namespace": ${namespace-name}
-  Egress:
-    - To:
-        - namespaceSelector:
-            matchLabels:
-              "kubesphere.io/workspace": ${namespace-name}
+  policyTypes:
+  - ingress
+```
+
+Let's say I have two projects, A and B. They are in same workspace. If the A and B have not set annotation "kubesphere.io/network-isolate = enabled", they can access each other. The same thing goes for services. If project A set annotation "kubesphere.io/network-isolate = enabled", A cannot access services in B, unless A allows egress for services in B.
+
+## The Namespace NetworkPolicy resource 
+See the [Namespace NetworkPolicy](https://github.com/kubesphere/kubesphere/blob/master/pkg/apis/network/v1alpha1/namespacenetworkpolicy_types.go#L31) reference for a full definition of the resource.
+With the exception of this field `service`, the fields are the same as kubernetes NetworkPolicy.
+
+An example Namespace NetworkPolicy might look like this:
+```yaml
+apiVersion: network.kubesphere.io/v1alpha1
+kind: NamespaceNetworkPolicy
+metadata:
+  name: test-nsnp
+  namespace: default
+spec:
+  egress:
+    - to:
+      - service:
+          name: my-svc
+          namespace: my-project
 ```
 
-Beyond that, we also allow the bellowing traffic:
+The example policy contains a single rule, which matches traffic selected by a service.
```

Comment:
The example policy contains a single rule, which matches the traffic selected by a service.